common-features/cdn.md

```diff
@@ -41,7 +41,7 @@ Here are a few examples:
 
 <div class="skip-repl"></div>
 
-````CHTML
+````RAZOR
```

Comment:
Do we need to use CAPS LOCK for the language name? It's more convenient without it.

Reply:
I don't think we've discussed lower casing here. I tested `Razor` and it also seems to be working. However, I'd prefer to keep the capitalization `RAZOR` for consistency with the other languages. Plus, it is visualized like so regardless of how it is written in the documentation:
<img width="112" alt="image" src="https://github.com/user-attachments/assets/fccf67fb-e11a-49cc-9229-b48ce7507c10">

I'm not sure if this can be changed on our side. If som do you think we should request it?

Reply:
TBD on the meeting tomorrow.